src/main/java/org/concordion/internal/command/executeCommand/ExecuteCommandTableStrategy.java

```diff
@@ -0,0 +1,85 @@
+package org.concordion.internal.command.executeCommand;
+
+import org.concordion.api.*;
+import org.concordion.internal.ConcordionBuilder;
+import org.concordion.internal.Row;
+import org.concordion.internal.TableSupport;
+import org.concordion.internal.command.ExampleCommand;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by tim on 12/06/16.
+ */
+public class ExecuteCommandTableStrategy implements ExecuteCommandStrategy {
+
+    private final ExecuteCommand executeCommand;
+
+    public ExecuteCommandTableStrategy(ExecuteCommand executeCommand) {
+        this.executeCommand = executeCommand;
+    }
+
+    public void execute(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
+        TableSupport tableSupport = new TableSupport(commandCall);
+
+        // check if this table is in "example" mode.
+        if (hasExamples(tableSupport)) {
+            return;
+        }
+
+        Row[] detailRows = tableSupport.getDetailRows();
+        for (Row detailRow : detailRows) {
+            if (detailRow.getCells().length != tableSupport.getColumnCount()) {
+                throw new RuntimeException("The <table> 'execute' command only supports rows with an equal number of columns.");
+            }
+            commandCall.setElement(detailRow.getElement());
+            tableSupport.copyCommandCallsTo(detailRow);
+            commandCall.execute(evaluator, resultRecorder);
+        }
+    }
+
+    private boolean hasExamples(TableSupport tableSupport) {
+        // if any of the columns have the exampleName tag, then we are running as examples.
+        return getExampleColumn(tableSupport) >= 0;
+    }
+
+    private int getExampleColumn(TableSupport tableSupport) {
+        Row headerRow = tableSupport.getLastHeaderRow();
+        for (Element e : headerRow.getCells()) {
+            String exampleText = e.getAttributeValue("exampleName", ConcordionBuilder.NAMESPACE_CONCORDION_2007);
```

Comment:
Or we could change it to 'example' - but I think there was a good reason I didn't use that. Perhaps because it gave some strange interactions with the example command.